02-your_first_shiny_app.Rmd

```diff
@@ -19,4 +19,4 @@
   
 ```
 ```
-</details>
```

Comment:
Can you remove this file from the commit so that it doesn't overwrite the other one by Jerome? (Sorry I didn't get to this sooner!)